extensions/blocks/business-hours/business-hours.php

```diff
@@ -20,11 +68,11 @@
  *
  * @return string
  */
-function jetpack_business_hours_render( $attributes, $content ) {
```

Comment:
Since you are removing this parameter, you'll want to update the docblock as well.